pyproject.toml

```diff
@@ -11,14 +11,18 @@ repository = "https://github.com/qdrant/fastembed"
 keywords = ["vector", "embedding", "neural", "search", "qdrant", "sentence-transformers"]
 
 [tool.poetry.dependencies]
-python = ">=3.8.0,<3.12"
-onnx = "^1.11"
-onnxruntime = "^1.15"
-tqdm = "^4.65"
+python = ">=3.8.0,<3.13"
+onnx = "^1.15.0"
+onnxruntime = "^1.17.0"
+tqdm = "^4.66"
 requests = "^2.31"
-tokenizers = "^0.15.0"
-huggingface-hub = "0.19.4"
+tokenizers = "^0.15.1"
+huggingface-hub = "^0.20.3"
```

Comment:
Do we need to be this strict here?

Why not 

```
tokenizers = "^0.15"
huggingface-hub = "^0.20"
```

Reply:
It's the later versions which have been released and tested for 3.12 

e.g. in the case of tokenizers: This PR is part of the latest release and adds Windows tests for Python 3.12: https://github.com/huggingface/tokenizers/pull/1438/

I'd also prefer to more relaxed but don't want this to break for a specific dependency.

Reply:
So the minimal version which we support is 0.15.1?

Reply:
And our options will be either unlock the dependency with `^0.16` once it is released or replace it with `>=0.15.1,<1.0.0`